src/app/designationEditor/photoModal/photo.modal.js

```diff
@@ -20,14 +35,52 @@ class ModalInstanceCtrl {
     this.maxNumberOfPhotos = 6
   }
 
-  uploadComplete () {
-    // refresh photos (set timeout to give Adobe time to add to DAM)
-    this.$timeout(() => {
-      this.designationEditorService.getPhotos(this.designationNumber).then((response) => {
-        this.photos = response.data
-        this.uploading = false
-      }, angular.noop)
-    }, 3500)
+  uploadComplete (response) {
+    this.uploading = false
+    ++this.numProcessingPhotos
+
+    this.refreshPhotos(new URL(response.headers('location'), this.$window.location).pathname).then((photos) => {
```

Comment:
Don't we want just the protocol and domain for the second parameter (base)?

Reply:
Either way works. [MDN has examples](https://developer.mozilla.org/en-US/docs/Web/API/URL/URL#examples) with full URLs as the base. But I could change it to `this.$window.location.origin` if you want me to.

Reply:
Oh, I see one now: `new URL("/en-US/docs", "https://developer.mozilla.org/fr-FR/toto");`